Source/Fluxor/DependencyInjection/ServiceRegistration/EffectClassRegistration.cs

```diff
@@ -7,10 +7,11 @@ internal static class EffectClassRegistration
 	{
 		public static void Register(
 			IServiceCollection services,
-			IEnumerable<EffectClassInfo> effectClassInfos)
+			IEnumerable<EffectClassInfo> effectClassInfos,
+			FluxorOptions options)
```

Comment:
I would like to keep the `options` being passed in, even though we'll only use the `options.ServiceLifetime` property.

Reply:
can you clarify?  Do you want this to be ```options``` passed in or ```options.ServiceLifetime``` prop passed in?

Reply:
I like options, as you have it :)